### PR TITLE
Updated explenation for for classy classification

### PR DESCRIPTION
**Pull request by davidberenstein1957**
Reviewed and simplified description and code example

## Description
only provide one concise example + mention reference to huggingface and sentence-transformers

### Types of change
change to the documentation

## Checklist
- [X] I confirm that I have the right to submit this contribution under the project's MIT license.
- [X] I ran the tests, and all new and existing tests passed.
- [X] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
Sorry, could you update or redo this PR without the formatting changes to unrelated entries in `universe.json`?

**Comment by davidberenstein1957** [comment]
> Sorry, could you update or redo this PR without the formatting changes to unrelated entries in `universe.json`?

@adrianeboyd that's what I get for using formatting on change. I update the universe.json accordingly. That should be fine if you squash commits on merge right?

File: website/meta/universe.json [hunk 1]
```diff
@@ -2601,8 +2601,9 @@
...
+            "title": "Classy Classification",
+            "slogan": "Have you every struggled with needing a Spacy TextCategorizer but didn't have the time to train one from scratch?. Classy Classification is the way to go!",
+            "description": "Have you every struggled with needing a [Spacy TextCategorizer](https://spacy.io/api/textcategorizer) but didn't have the time to train one from scratch? Classy Classification is the way to go! For few-shot classification using [sentence-transformers](https://github.com/UKPLab/sentence-transformers) or [spaCy models](https://spacy.io/usage/models), provide a dictionary with labels and examples, or just provide a list of labels for zero shot-classification with [Hugginface zero-shot classifiers](https://huggingface.co/models?pipeline_tag=zero-shot-classification).",
```
Comment (adrianeboyd) [inline website/meta/universe.json:2606]:
```suggestion
            "slogan": "Have you ever struggled with needing a spaCy TextCategorizer but didn't have the time to train one from scratch?. Classy Classification is the way to go!",
            "description": "Have you ever struggled with needing a [spaCy TextCategorizer](https://spacy.io/api/textcategorizer) but didn't have the time to train one from scratch? Classy Classification is the way to go! For few-shot classification using [sentence-transformers](https://github.com/UKPLab/sentence-transformers) or [spaCy models](https://spacy.io/usage/models), provide a dictionary with labels and examples, or just provide a list of labels for zero shot-classification with [Hugginface zero-shot classifiers](https://huggingface.co/models?pipeline_tag=zero-shot-classification).",
```

**Comment by adrianeboyd** [comment]
Thanks! Just made a suggestion related to some minor formatting/typos.

**Comment by davidberenstein1957** [comment]
@adrianeboyd I accepted the commit and resolved another small typo. Should be good to go now.

**Pull request merged**